package.json

```diff
@@ -45,7 +45,7 @@
     "node": ">=8.3.0"
   },
   "dependencies": {
-    "postcss-cli": "^10.0.0",
+    "jest-environment-jsdom": "^29.5.0",
```

Comment:
This should be a dev dependency, since it should only be necessary for tests

Reply:
Oh right, thanks!